extensions/amp-story/1.0/amp-story-localization-service.js

```diff
@@ -58,9 +49,11 @@ export function localizeTemplate(template, context) {
       localizationService
         .getLocalizedStringAsync(el.getAttribute('i-amphtml-i18n-text-content'))
         .then((str) =>
-          vsync.mutatePromise(() => {
-            el.textContent = str;
-          })
+          template.isConnected
```

Comment:
We don't need to mutate if the template is not connected because the DOM is only updated when it's connected